test/PowerShellEditorServices.Test/Extensions/ExtensionServiceTests.cs

```diff
@@ -1,255 +1,266 @@
-        {
-            await this.extensionEventQueue.EnqueueAsync(
-                new Tuple<EventType, EditorCommand>(EventType.Add, e));
-        }
-
-        private async void ExtensionService_ExtensionUpdated(object sender, EditorCommand e)
-        {
-            await this.extensionEventQueue.EnqueueAsync(
-                new Tuple<EventType, EditorCommand>(EventType.Update, e));
-        }
-
-        private async void ExtensionService_ExtensionRemoved(object sender, EditorCommand e)
-        {
-            await this.extensionEventQueue.EnqueueAsync(
-                new Tuple<EventType, EditorCommand>(EventType.Remove, e));
-        }
-    }
-
-    public class TestEditorOperations : IEditorOperations
-    {
-
-        public string GetWorkspacePath()
-        {
-            throw new NotImplementedException();
-        }
-
-        public string GetWorkspaceRelativePath(string filePath)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task NewFileAsync()
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task OpenFileAsync(string filePath)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task OpenFileAsync(string filePath, bool preview)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task CloseFileAsync(string filePath)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task SaveFileAsync(string filePath)
-        {
-            return SaveFileAsync(filePath, null);
-        }
-
-        public Task SaveFileAsync(string filePath, string newSavePath)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task InsertTextAsync(string filePath, string text, BufferRange insertRange)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task SetSelectionAsync(BufferRange selectionRange)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task<EditorContext> GetEditorContextAsync()
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task ShowInformationMessageAsync(string message)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task ShowErrorMessageAsync(string message)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task ShowWarningMessageAsync(string message)
-        {
-            throw new NotImplementedException();
-        }
-
-        public Task SetStatusBarMessageAsync(string message, int? timeout)
-        {
-            throw new NotImplementedException();
-        }
-    }
-}
+// //
+// // Copyright (c) Microsoft. All rights reserved.
+// // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// //
+
+// using Microsoft.Extensions.Logging.Abstractions;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4993607)

Reply:
These were flaky and need to be revisited...